js/src/jit-test/tests/basic/bug1875363.js

```diff
@@ -1,4 +1,4 @@
-// |jit-test| --fuzzing-safe; --baseline-eager; --arm-hwcap=vfp
+// |jit-test| --fuzzing-safe; --baseline-eager; --arm-hwcap=vfp; skip-if: getBuildConfiguration('pbl')
```

Comment:
We should probably add a comment to explain why we skip this test in when using PBL

Reply:
Done!